### DISCUSSION TITLE
branch for tag creation or push event ?

### DISCUSSION THREAD

**Discussion by kapouer** [community/community#24859] [Apps, API and Webhooks]
<p>Hi,</p>
<p>webhooks events Push et Create don’t seem to carry which branch the tag was set on ?</p>
<p>Note that i’m often doing <code>git tag xxx -sm yyy</code>.</p>

**Comment by kapouer** [comment, marked as answer]
<p>I just found the answer:</p>
<p>A <strong>tag</strong> is a <strong>pointer to a commit</strong> , and <strong>commits exist independently of branches</strong>.</p>
<p>It is important to understand that <strong>tags have no <em>direct</em> relationship with branches</strong> - they <strong>only ever identify a <em>commit</em></strong>.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/kapouer">@kapouer</a>, thank you for coming back to share your solution! We’re looking forward to seeing you around!</p>
